sky/clouds/runpod.py

```diff
@@ -0,0 +1,283 @@
+""" RunPod Cloud. """
+
+import json
+import typing
+from typing import Dict, Iterator, List, Optional, Tuple
+
+from sky import clouds
+from sky import status_lib
+from sky.clouds import service_catalog
+import sky.skylet.providers.runpod.rp_helper as runpod_api
+
+if typing.TYPE_CHECKING:
+    from sky import resources as resources_lib  # Renaming to avoid shadowing variables.
+
+_CREDENTIAL_FILES = [
+    'credentials.toml',
+]
+
+
+@clouds.CLOUD_REGISTRY.register
+class RunPod(clouds.Cloud):
+    """ RunPod GPU Cloud
+
+    _REPR | The string representation for the RunPod GPU cloud object.
+    """
+    _REPR = 'RunPod'
+    _CLOUD_UNSUPPORTED_FEATURES = {
+        clouds.CloudImplementationFeatures.STOP: 'FluffyCloud does not support stopping VMs.',
+        clouds.CloudImplementationFeatures.AUTOSTOP: 'FluffyCloud does not support stopping VMs.',
```

Comment:
Should we update the message?

Also, it seems `STOP` is supported by runpod, according to the status mapping below. Should we remove at least the `clouds.CloudImplementationFeatures.STOP` from the dict?